pkgs/by-name/ag/agate/package.nix

```diff
@@ -30,7 +30,7 @@ rustPlatform.buildRustPackage rec {
     [ openssl ]
     ++ lib.optionals stdenv.hostPlatform.isDarwin [
       libiconv
-      darwin.apple_sdk.frameworks.Security
+      apple-sdk_11
```

Comment:
This does not seem correct: the original package didn’t override the SDK at all, and `apple-sdk_11` is not the default on `x86_64-darwin`. If you want the default SDK, you don’t need to specify anything; non‐overridden `darwin.apple_sdk.frameworks.*` inputs in `by-name` should simply be dropped entirely. `libiconv` too.

Most likely the OpenSSL dependency is not used on Darwin, so it should instead be `buildInputs = lib.optionals (!stdenv.hostPlatform.isDarwin) [ openssl ];`, but as that was a pre‐existing problem it’s not a blocker for this PR.

Reply:
Oh! I misunderstood.

Changed just this one, if it looks good I'll go do the rest.

Reply:
Looks good to me. Any time we’re conditionalizing `openssl` on not‐Darwin we’ll need to check it still builds (some things *do* want OpenSSL on Darwin, especially if they’re not already including the `Security` framework), but just removing the frameworks and `libiconv` should be the way to go for anything that doesn’t specifically override the version currently.